plugins/user_quota/web_client/js/userQuota.js

```diff
@@ -334,11 +346,11 @@ girder.userQuota = {
      *                    for none. */
     valueAndUnitsToSize: function (sizeValue, sizeUnits) {
```

Comment:
(Maybe not get rid of the function, but at least make it more readable by removing the regexp and the looping...)